docs/src/usage/usage.rst

```diff
@@ -1,21 +1,11 @@
-****************
-Using Opaque SQL
-****************
-
-Setup
-#####
-Opaque SQL needs three Spark properties to be set:
-
-- ``spark.executor.instances``
-- ``spark.task.maxFailures``
--  ``spark.driver.defaultJavaOptions="-Dscala.color"`` (if running the gRPC listener)
-
-These properties can be be set in a custom configuration file, the default being located at ``${SPARK_HOME}/conf/spark-defaults.conf``, or as a ``spark-submit`` or ``spark-shell`` argument: ``--conf <key>=<value>``.
+**************************
+Querying with Spark Driver
```

Comment:
Suggestion: "Query submission via the Spark driver".

Also: I don't think the driver is defined anywhere? Might be good to include a diagram in the front page, for example from the Opaque paper.